### ISSUE TITLE
Bar graph it slightly off screen

### ISSUE THREAD

**Issue by ddaddy** [hankinsoft/AppWage#9]
No matter what size I size the app, the last bar of the bar chart is always off screen.

<img width="322" alt="screen shot 2016-11-08 at 10 35 23" src="https://cloud.githubusercontent.com/assets/1425808/20095951/96e54f20-a59f-11e6-94f3-34126d8d17a6.png">

**Comment by hankinsoft** [comment]
Odd, I'm seeing this today as well. I'll take a look soon.

**Comment by hankinsoft** [comment]
FYI this seems to have gone away. I'm sure there is still an underling issue and it will probably happen again though. If you are no longer seeing it, I'm going to close the issue for the time being.

**Comment by ddaddy** [comment]
Yes, it has strangely gone away for me too. Without changing anything.
Very odd.
Maybe it has something to do with the number of days in a month? (Just a guess).

**Comment by ddaddy** [comment]
This seems to have come back since yesterday. I wonder if it has something to do with the number of days in the month?

<img width="195" alt="screen shot 2017-11-02 at 13 05 09" src="https://user-images.githubusercontent.com/1425808/32327517-adb20c6c-bfce-11e7-9f63-620c3ec7f62e.png">

**Comment by hankinsoft** [comment]
Thanks! I'm guessing its based on the month, as this issue was open last November/December. I'm not sure when I'll get a chance to look into this, but pull requests are always welcome :D
